src/main/java/org/betonquest/betonquest/quest/event/journal/JournalEvent.java

```diff
@@ -41,11 +41,16 @@ public JournalEvent(final BetonQuest betonQuest, final JournalChanger journalCha
     }
 
     @Override
+    @SuppressWarnings("PMD.EmptyCatchBlock")
     public void execute(final Profile profile) {
         final PlayerData playerData = betonQuest.getOfflinePlayerData(profile);
         final Journal journal = playerData.getJournal();
         journalChanger.changeJournal(journal);
         journal.update();
-        notificationSender.sendNotification(profile.getOnlineProfile());
+        try {
+            notificationSender.sendNotification(profile.getOnlineProfile());
+        } catch (final IllegalStateException exception) {
+            // ignore
+        }
```

Comment:
This is just not a proper solution. The issue here lies a bit deeper and is bad API design. It would be better if one could fetch an `Optional<OnlinePlayer>` and then call the NotificationSender via `.ifPresent(notificationSender::sendNotification)`.

Reply:
and if you catch such an exception, always log them as debug. But i also think that you should check for if the player is online